src/systemTest/java/com/flightstats/hub/callback/CallbackResource.java

```diff
@@ -13,23 +14,39 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Predicate;
 
 @Slf4j
 @Path("/callback")
 public class CallbackResource {
 
-    private CallbackCache callbackCache;
+    private final CallbackCache callbackCache;
+    private final List<Predicate<WebhookCallbackRequest>> responseOverrides;
```

Comment:
Wanted to note that in #1156 this file is removed, I am (just assuming) that the intended use here will be moved to the external callback server in our k8s tests. You may have to coordinate with @manjula-chikkanagappa  to confirm, this error cache functionality looks good generally and is something that has not been added to that repo yet, which you can PR or let me know if you are busy with other stuff and I will add it

Reply:
Yeah agreed.  I think this is something @manjula-chikkanagappa is starting to add on her side.  Pinged her to get her thoughts on it.